learning_paths/apps.py

```diff
@@ -13,6 +13,8 @@ class LearningPathsConfig(AppConfig):
 
     name = "learning_paths"
 
+    default_auto_field = "django.db.models.AutoField"
```

Comment:
This is [the default value in Django](https://docs.djangoproject.com/en/4.2/ref/settings/#std-setting-DEFAULT_AUTO_FIELD). Why do we need it?

Reply:
Removed in 8a2d504.

Reply:
@Agrendalath There are a lot of warning when [running migration commands](https://github.com/open-craft/learning-paths-plugin/actions/runs/14196731514/job/39773525489?pr=11#step:6:27). Although, I can't remember if I saw the when generating the migrations.

I added it when I was trying to fix all the CI errors. Might have been an overkill.